pkg/controllers/networkpolicy/policy_controller.go

```diff
@@ -166,7 +165,7 @@ func NewPolicyController(k8sClientset *kubernetes.Clientset, calicoClient *clien
 		},
 	}, cache.Indexers{})
 
-	return &PolicyController{indexer, informer, ccache, calicoClient, k8sClientset}
```

Comment:
Interesting, the controller doesn't need the client I assume?

Reply:
yeah, looks that way - was only referencing the extensions from clientset.